bigtable-hbase-1.x-parent/bigtable-hbase-1.x-mapreduce/README.md

```diff
@@ -2,51 +2,89 @@
 2. Download service account credentials json from Google Cloud Console.
 3. Submit the job using your edge node's hadoop installation. 
    ```bash
-   # Export to SequenceFiles
-   GOOGLE_APPLICATION_CREDENTIALS=path/to/service-account.json \
-   hadoop jar bigtable-hbase-1.x-mapreduce-2.0.0-alpha1-hadoop.jar \
-       export-table \
-       -Dgoogle.bigtable.project.id=<project-id> \
-       -Dgoogle.bigtable.instance.id=<instance-id> \
-       <table-id> \
-       <outputdir>
+    # Export to SequenceFiles
+    GOOGLE_APPLICATION_CREDENTIALS=path/to/service-account.json \
+    hadoop jar bigtable-hbase-1.x-mapreduce-2.5.0-shaded-byo-hadoop.jar \
+        export-table \
+        -Dgoogle.bigtable.project.id=<project-id> \
+        -Dgoogle.bigtable.instance.id=<instance-id> \
+        <table-id> \
+        <outputdir>
    
-   # Import from SequenceFiles
-      GOOGLE_APPLICATION_CREDENTIALS=path/to/service-account.json \
-      hadoop jar bigtable-hbase-1.x-mapreduce-2.0.0-alpha1-hadoop.jar \
-          import-table \
-          -Dgoogle.bigtable.project.id=<project-id> \
-          -Dgoogle.bigtable.instance.id=<instance-id> \
-          <table-id> \
-          <inputdir>
+    # Import from SequenceFiles
+    GOOGLE_APPLICATION_CREDENTIALS=path/to/service-account.json \
+    hadoop jar bigtable-hbase-1.x-mapreduce-2.5.0-shaded-byo-hadoop.jar \
+        import-table \
+        -Dgoogle.bigtable.project.id=<project-id> \
+        -Dgoogle.bigtable.instance.id=<instance-id> \
+        <table-id> \
+        <inputdir>
+   
+    # Import from HBase snapshot
+    GOOGLE_APPLICATION_CREDENTIALS=path/to/service-account.json \
+    hadoop jar bigtable-hbase-1.x-mapreduce-2.5.0-shaded-byo-hadoop.jar \
+        import-snapshot \
+        -Dgoogle.bigtable.project.id=<project-id> \
+        -Dgoogle.bigtable.instance.id=<instance-id> \
+        <snapshot-name> \
+        <snapshot-dir> \
+        <table-id> \
+        <tmp-dir>
+   
+    # HashTable on HBase
+    GOOGLE_APPLICATION_CREDENTIALS=path/to/service-account.json \
+    hadoop jar bigtable-hbase-1.x-mapreduce-2.5.0-shaded-byo-hadoop.jar \
+        hash-table \
+        -Dhbase.zookeeper.quorum=<source-zk-quorum> \
+        <source-table-id> \
+        <hash-outputdir-hbase>
    
-   # Import from HBase snapshot
-      GOOGLE_APPLICATION_CREDENTIALS=path/to/service-account.json \
-      hadoop jar bigtable-hbase-1.x-mapreduce-2.0.0-alpha1-hadoop.jar \
-          import-snapshot \
-          -Dgoogle.bigtable.project.id=<project-id> \
-          -Dgoogle.bigtable.instance.id=<instance-id> \
-          <snapshot-name> \
-          <snapshot-dir> \
-          <table-id> \
-          <tmp-dir>
+    # SyncTable on Bigtable (dryrun enabled by default)
+    GOOGLE_APPLICATION_CREDENTIALS=path/to/service-account.json \
+    hadoop jar bigtable-hbase-1.x-mapreduce-2.5.0-shaded-byo-hadoop.jar \
+        sync-table \
+        --sourcezkcluster=<source-zk-quorum> \
```

Comment:
why change styles of system properties already in use?

Reply:
good question, these are non-conventional, but we're following HBase upstream [HashTable.java#L616-L758](https://github.com/apache/hbase/blob/master/hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/HashTable.java#L616-L758) and  [SyncTable.java#L753-L854](https://github.com/apache/hbase/blob/master/hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/SyncTable.java#L753-L854) conventions with options and args.

Reply:
its a shame that I didn't follow these to begin with :(